zerver/models.py

```diff
@@ -1414,6 +1414,7 @@ class UserBaseSettings(models.Model):
     dense_mode: bool = models.BooleanField(default=True)
     fluid_layout_width: bool = models.BooleanField(default=False)
     high_contrast_mode: bool = models.BooleanField(default=False)
+    send_private_message_on_invitee_signup: bool = models.BooleanField(default=True)
```

Comment:
Not sure of the variable name here since the user receives private message. We should change it as per the feedback given in the above comment.